lib/linguist/languages.yml

```diff
@@ -41,6 +41,14 @@ AGS Script:
   tm_scope: source.c++
   ace_mode: c_cpp
 
+AMPL:
+  type: programming
+  color: "#00008B"
+  extensions:
+  - .ampl
+  tm_scope: none
```

Comment:
You'll need to change this line to `tm_scope: source.ampl` to use the grammar ;)